roles/sap_general_preconfigure/tasks/RedHat/generic/check-dns-name-resolution.yml

```diff
@@ -1,17 +1,23 @@
 ---
 
-- name: Check dns forwarding settings
+- name: Check forward dns record (A)
   ansible.builtin.shell: test "$(dig {{ sap_general_preconfigure_hostname }}.{{ sap_general_preconfigure_domain }} +short)" = "{{ sap_general_preconfigure_ip }}"
   changed_when: false
-  ignore_errors: true
+ #ignore_errors: true
+  register: __sap_general_preconfigure_register_dns_test_a
+  failed_when: __sap_general_preconfigure_register_dns_test_a.rc != 0
 
 ### BUG: dig does not use search path in resolv.con on PPCle
 - name: Check resolv.conf settings
   ansible.builtin.shell: test "$(dig {{ sap_general_preconfigure_hostname }} +search +short)" = "{{ sap_general_preconfigure_ip }}"
   changed_when: false
-  ignore_errors: true
+#ignore_errors: true
+  register: __sap_general_preconfigure_register_dns_test_resolvconf
```

Comment:
We should maybe change the variable name to `__sap_general_preconfigure_register_dns_test_a_searchlist`